tests/testthat/test_aggregate_sex.R

```diff
@@ -0,0 +1,51 @@
+testthat::context("aggregate_sex() tests")
+
+# set up standard input data.table
+input_dt <- data.table::CJ(location = "France", year = 2010:2020,
```

Comment:
From the "Setup and Teardown" section of https://www.tidyverse.org/blog/2017/12/testthat-2-0-0/, would it be useful to have a single place for test data that is used across tests?

Reply:
Yeah I wasn't sure of the best way to do this, I also need test data that is mis-formatted. Would that go there too? Or should I add known errors to the test datasets?

Reply:
Is there enough consistency in what data we'd want to be using for a given test?

Reply:
Lets revisit this later